### DISCUSSION TITLE
Weblate for translation

### DISCUSSION THREAD

**Discussion by xldrkp** [singlesourcepub/community#33] [Tools]
#FYI Around Christmas I took some hours to set up [weblate](https://weblate.org/en/) and integrated it into a CI/CD workflow with GitLab. Works like a charm although translation of Markdown documents needs some workarounds. I can report more if anyone's interested. @mrchristian @VictorVenema

**Comment by VictorVenema** [comment]
I recently discovered a book on historical climate measurements written over a century ago in German by none other than Gustav Hellmann. I presume that means that it is not longer copy-righted and we would be allowed to publish a translation. But I have not read the book yet, and it is 60 pages, so I am not committing to doing it yet. 
https://www.worldcat.org/title/beitrage-zur-erfindungsgeschichte-meteorologischer-instrumente/oclc/247727764

**Reply by VictorVenema** [reply]
The book was published by the Prussian Academy of Science, which is now the Berlin Brandenburg Academy of Sciences and they send me a digital copy. That is one step less. Now we only have to do the OCR. It is probably not the historically most important book, but it may be a good test case, it has all the problems you can think off: A massive amount of foot notes, tables, figures, untranslated quotes in French and Italian, references.

**Comment by comradekingu** [comment]
@VictorVenema The Debian Handbook has a very nice Weblate setup for translating what amounts to a book.
